hdf5-vol-daos.spec

```diff
@@ -0,0 +1,170 @@
+%global with_mpich 1
+%global with_openmpi3 1
+
+%if %{with_mpich}
+%global mpi_list mpich
+%endif
+%if %{with_openmpi3}
+%global mpi_list %{?mpi_list} openmpi3
+%endif
+
+%global test_commit 31660ab19352049cf59f98b1fba498b3e93dade5
+
+%if (0%{?suse_version} >= 1500)
+%global module_load() if [ "%{1}" == "openmpi3" ]; then module load gnu-openmpi; else module load gnu-%{1}; fi
+%else
+%global module_load() module load mpi/%{1}-%{_arch}
+%endif
+
+Name:    hdf5-vol-daos
+Version: 0.1
+Release: b324b90d4554e211976174b9a3acdba61c184248
+Summary: A Multi-purpose, Application-Centric, Scalable I/O Proxy Application
+
+License: GPL
+URL: https://portal.hdfgroup.org/display/HDF5/HDF5
+Source0: https://github.com/HDFGroup/vol-daos/archive/%{release}.tar.gz
+Source1: https://github.com/HDFGroup/vol-tests/archive/%{test_commit}.tar.gz
```

Comment:
With this implementation shouldn't the `daos-vol-master@b324b90d455.zip` and `vol-tests-master@31660ab1935.zip` files be removed from this commit?